### PR TITLE
Project improvements

### PR DESCRIPTION
**Pull request by ricknox**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-const dotenv = require('dotenv');
-
-dotenv.config();
-
-require('./src/bot'); // @todo: Find a better way to do this.
```
Comment (Pierstoval) [inline index.js]: I'm ok to remove it, but as an entrypoint is often called "index" or "main", why not rename `bot.js` to `main.js`?

Comment (ricknox) [reply index.js]: True, but pm2 doesn't care about that so imo. it's just relics and overhead to have a main file. I do agree to rename `bot.js` to `main.js`. Fixed in 4bd62f07261959831c5128e4f836cf56b33dde50.

Comment (Pierstoval) [reply index.js]: I think "main" is a standard everywhere, since `main()` function in C and tons of other languages, hence my suggestion 😉

**Comment by ricknox** [comment]
Squashed and merged in dfde769d7f34df159b22b02f69e203a53049a27d.

**Pull request closed without merging**